ui/src/lib/utils.ts

```diff
@@ -24,9 +24,34 @@ export function cn(...inputs: ClassValue[]) {
   return twMerge(clsx(inputs));
 }
 
-export function getOrtDateTimeString(timestamp: string) {
-  return new Date(timestamp).toLocaleString(navigator.language, {
+export function formatDate(
+  timestamp: string,
+  locales: Intl.LocalesArgument = navigator.language
+) {
+  return new Date(timestamp).toLocaleString(locales, {
     day: '2-digit',
     month: '2-digit',
+    year: 'numeric',
+    hour: '2-digit',
+    minute: '2-digit',
+    second: '2-digit',
+  });
+}
+
+if (import.meta.vitest) {
+  const { it, expect } = import.meta.vitest;
+
+  it('formatDate', () => {
+    expect(formatDate('2024-06-11T13:07:45Z', 'en-US')).toBe(
+      '06/11/2024, 03:07:45 PM'
+    );
+
+    expect(formatDate('2024-06-11T13:07:45Z', 'de-DE')).toBe(
+      '11.06.2024, 15:07:45'
+    );
+
+    expect(formatDate('2024-06-11T13:07:45Z', 'fi-FI')).toBe(
+      '11.06.2024 klo 15.07.45'
```

Comment:
@lamppu, from your screenshot at https://github.com/eclipse-apoapsis/ort-server/pull/1238 I see that a Finnish date / time without "klo" is used. Are you using a different locale than `fi-FI` there? because otherwise I would have no idea how to get rid of the "klo" here.

Reply:
Ok, um weird, it is `fi-FI`

Reply:
Looks like it could be browser specific, for example in Firefox it doesn't add the "klo" but in Chrome it does

Reply:
Oh, wow. Thanks for checking!